### ISSUE TITLE
`FileWriter` cannot find MacroTools

### ISSUE THREAD

**Issue by MSeeker1340** [malmaud/TensorFlow.jl#446]
```
julia> using TensorFlow.summary: FileWriter

julia> w = FileWriter(".")
ERROR: On worker 2:
LoadError: ArgumentError: Package MacroTools not found in current path:
- Run `Pkg.add("MacroTools")` to install the MacroTools package.

require at ./loading.jl:817
include at ./boot.jl:317 [inlined]
include_relative at ./loading.jl:1038
include at ./sysimg.jl:29
include at ./client.jl:388
#3 at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/TensorFlow.jl:154
#116 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/process_messages.jl:276
run_work_thunk at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/process_messages.jl:56
run_work_thunk at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/process_messages.jl:65
#102 at ./task.jl:259
in expression starting at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/py.jl:2
Stacktrace:
 [1] #remotecall_wait#154(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Function, ::Distributed.Worker, ::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/remotecall.jl:407
 [2] remotecall_wait(::Function, ::Distributed.Worker, ::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/remotecall.jl:398
 [3] #remotecall_wait#157(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Function, ::Int64, ::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/remotecall.jl:419
 [4] remotecall_wait(::Function, ::Int64, ::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Distributed/src/remotecall.jl:419
 [5] top-level scope at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/TensorFlow.jl:153
 [6] eval at ./boot.jl:319 [inlined]
 [7] eval at ./sysimg.jl:68 [inlined]
 [8] #load_python_process#1(::Bool, ::Function) at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/TensorFlow.jl:150
 [9] load_python_process at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/TensorFlow.jl:143 [inlined]
 [10] macro expansion at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/TensorFlow.jl:181 [inlined]
 [11] #FileWriter#1(::TensorFlow.Graph, ::Type, ::String) at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/summary_writer.jl:34
 [12] FileWriter(::String) at /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/summary_writer.jl:32
 [13] top-level scope at none:0
```

I have MacroTools in my Manifest.toml but still got this error. The problem is fixed if I explicitly `Pkg.add("MacroTools")`.

Version Info:

```
----------------
Library Versions
----------------
Trying to evaluate ENV["TF_USE_GPU"] but got error: KeyError("TF_USE_GPU")
Trying to evaluate ENV["LIBTENSORFLOW"] but got error: KeyError("LIBTENSORFLOW")

tf_version(kind=:backend) = 1.10.0
Trying to evaluate tf_version(kind=:python) but got error: RemoteException(2, CapturedException(LoadError("/home/mseeker/.julia/packages/TensorFlow/A6TdG/src/py.jl", 2, ArgumentError("Package MacroTools not found in current path:\n- Run `Pkg.add(\"MacroTools\")` to install the MacroTools package.\n")), Any[(require at loading.jl:817, 1), (include at boot.jl:317 [inlined], 1), (include_relative at loading.jl:1038, 1), (include at sysimg.jl:29, 1), (include at client.jl:388, 1), (#3 at TensorFlow.jl:154, 1), (#116 at process_messages.jl:276, 1), (run_work_thunk at process_messages.jl:56, 1), (run_work_thunk at process_messages.jl:65, 1), (#102 at task.jl:259, 1)]))
tf_version(kind=:julia) = 0.10.2

-------------
Python Status
-------------
PyCall.conda = false
Trying to evaluate ENV["PYTHON"] but got error: KeyError("PYTHON")
PyCall.PYTHONHOME = /home/mseeker/anaconda3:/home/mseeker/anaconda3
String(read(#= /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/version.jl:104 =# @cmd("pip --version"))) = pip 10.0.1 from /home/mseeker/anaconda3/lib/python3.6/site-packages/pip (python 3.6)

Trying to evaluate String(read(#= /home/mseeker/.julia/packages/TensorFlow/A6TdG/src/version.jl:105 =# @cmd("pip3 --version"))) but got error: Base.IOError("could not spawn `pip3 --version`: no such file or directory (ENOENT)", -2)

------------
Julia Status
------------
Julia Version 1.0.0
Commit 5d4eaca0c9 (2018-08-08 20:58 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```

**Comment by malmaud** [comment]
Same issue as https://github.com/malmaud/TensorFlow.jl/issues/439

**Comment by oxinabox** [comment]
solved in #493
